dll/win32/shell32/dialogs/filetypes.cpp

```diff
@@ -296,12 +295,6 @@ GetTypeName(PFILE_TYPE_ENTRY Entry, PFILE_TYPE_GLOBALS pG)
             {
                 StringCchCopyW(Entry->FileDescription, _countof(Entry->FileDescription), fi.szTypeName);
             }
-            else
```

Comment:
It can't fail for `SHGFI_TYPENAME | SHGFI_USEFILEATTRIBUTES`. If there is no custom friendly name for said extension, you end up in `("%s File", pszExtension)` -> `"EXT File"`.